schemas/com.snowplowanalytics.snowplow/media_player_ping_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a media ping event sent in regular intervals during playback.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_ping_event",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "label": {
```

Comment:
Might be nice if we can re-use this for percentage based progress as well. Do we want to have units somewhere?

Reply:
In PR #1283, I added a schema for the `percent_progress` events as well.

We store the current progress using the `currentTime` and `percentProgress` properties in the [`media_player` entity schema](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow/media_player/jsonschema/1-0-0#L47).

Is that what you mean by units or did I misunderstand?

Reply:
We addressed this in the PR for events.